plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/expression/debug/TreeView.kt

```diff
@@ -64,20 +71,73 @@ class PlaintextLeaf(val text: String) : TreeView {
     }
 }
 
+
+class OperatorNode(val left: TreeView, val operator: String, val right: TreeView) : TreeView {
```

Comment:
Small note: the `operator` name is a reserved keyword in Kotlin, you could change it in something like `operatorSymbol`.